server.js

```diff
@@ -140,9 +146,9 @@ function new_client(id, opt, cb) {
 }
 
 module.exports = function(opt) {
-    opt = opt || {};
+    options = opt || {};
```

Comment:
If you're more comfortable with it, we can pass `opts` as a 4th parameter to `maybe_bounce` on the [new line 245](https://github.com/CWSpear/localtunnel-server/blob/subdomain/server.js#L245)